performance/scaling_data_ingestion.rst

```diff
@@ -16,8 +16,8 @@ When processing an INSERT, Citus first finds the right shard placements based on
 
     -- Set up a distributed table containing counters
     CREATE TABLE counters (c_key text, c_date date, c_value int, primary key (c_key, c_date));
-    SELECT master_create_distributed_table('counters', 'c_key', 'hash');
-    SELECT master_create_worker_shards('counters', 128, 2);
+    SET citus.shard_count = 128;
```

Comment:
I guess an advantage would be lock granularity when replication factor > 1 and doing non-commutative operation, but that seems besides the point here so the default seems reasonable enough.